server/integration/testsuite/src/test/java/org/infinispan/server/test/rollingupgrades/HotRodRollingUpgradesIT.java

```diff
@@ -33,30 +33,7 @@
  */
 @RunWith(Arquillian.class)
 @Category({RollingUpgrades.class})
```

Comment:
It seems the `RollingUpgrades` category is not run by default, I think it should be. Maven/Arquillian could be used to provide the old server